core/typedefs.h

```diff
@@ -97,14 +97,15 @@ T *_nullptr() {
 #undef CLAMP // override standard definition
 #undef Error
 #undef OK
+#undef CONNECT_DEFERRED // override from Windows SDK, clases with Object enum
```

Comment:
clas*h*es ;)

You can also remove the style changes as they're already in master.